lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -561,15 +553,11 @@ export abstract class ClientApplication {
                                 subErrorCode: error.subError,
                                 success: false
                             });
-                            atbcMeasurement.flushMeasurement();
                             throw error;
                         });
                     this.hybridAuthCodeResponses.set(hybridAuthCode, response);
                 } else {
                     this.logger.verbose("Existing acquireTokenByCode request found", request.correlationId);
-                    atbcMeasurement.endMeasurement({
```

Comment:
Why was this removed?

Reply:
We don't need to end measurements before discarding them because no events are emitted in that case.